connector/src/test/java/it/eng/connector/integration/datatransfer/DataTransferProcessRequestedIntegrationTest.java

```diff
@@ -148,6 +155,83 @@ public void initiateDataTransfer_already_requested() throws Exception {
        	String response = result.andReturn().getResponse().getContentAsString();
       	TransferError transferError = TransferSerializer.deserializeProtocol(response, TransferError.class);
       	assertNotNull(transferError);
+      	
+      	// cleanup
+    	transferProcessRepository.deleteById(transferProcessRequested.getId());
+	}
+	
+	@Test
+    @WithUserDetails(TestUtil.CONNECTOR_USER)
+    public void initiateDataTransfer_wrongDatasetFormat() throws Exception {
+		// finalized contract negotiation
+				Permission permission = Permission.Builder.newInstance()
+		    			.action(Action.USE)
+		    			.constraint(Arrays.asList(Constraint.Builder.newInstance()
+		    					.leftOperand(LeftOperand.COUNT)
+		    					.operator(Operator.LTEQ)
+		    					.rightOperand("5")
+		    					.build()))
+		    			.build();
+				Agreement agreement = Agreement.Builder.newInstance()
+		    			.assignee("assignee")
+		    			.assigner("assigner")
+		    			.target("test_dataset")
+		    			.permission(Arrays.asList(permission))
+		    			.build();
+		    	agreementRepository.save(agreement);
+		    	
+		    	// finalized contract negotiation
+		    	ContractNegotiation contractNegotiationFinalized = ContractNegotiation.Builder.newInstance()
+		    			.consumerPid(createNewId())
```

Comment:
.consumerPid("temporary_consumer_pid")
and line 
195 should be like this one?

Reply:
yep, because we are creating new pids for Data Transfer